### DISCUSSION TITLE
Use of Pint in OMAS

### DISCUSSION THREAD

**Discussion by orso82** [gafusion/omas#264] [Polls]
Does anyone use the units functionality in OMAS, provided by the `pint` packge?
See for example https://gafusion.github.io/omas/auto_examples/omas_units.html#sphx-glr-auto-examples-omas-units-py

We not not use it anywhere in OMFIT.

This is related to https://github.com/gafusion/omas/issues/263

**Comment by orso82** [comment]
Pinging @gafusion/omas_developers to please let us know if anyone uses the units functionality in OMAS?

**Comment by kripnerl** [comment]
We do not use it; however, I found the possibility of using it pretty cool and possibly usable in the future.

**Reply by hassec** [reply]
I'd say we are currently in the same situation. We are starting to use OMAS in more and more scenarios, and seeing the support for `pint` was one of the things we valued when we first considered its use.

**Reply by orso82** [reply]
Ok, so we should wait before removing that functionality :)

I think we ended up not using the automated units translation because 1) none of our existing code was explicitly carrying units around, and 2) once things are in IMAS then the units are consistent.

Anyways, we can give it some more time and revisit at a later time.
